pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/dedicated-cluster-reader.yaml

```diff
@@ -0,0 +1,139 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  annotations:
+    authorization.openshift.io/system-only: 'true'
```

Comment:
Do you know what this annotation is doing? You don't include it in dedicated-cluster-admin.

Reply:
@enj said this was old.  Removing.